book/art/tvu-runtime.bob

```diff
@@ -0,0 +1,8 @@
+.------------.    .-----------.    .-------------.    .--------------.    
+| PoH verify +--->| sigverify +--->| lock memory +--->| validate fee +--->
+`------------`    `-----------`    `-------------`    `--------------`    
```

Comment:
Can you merge these into one diagram? A merged diagram should make it obvious where we can optimize the pipeline when the a validator is acting as a slot leader.